packages/gatsby/rollup.config.js

```diff
@@ -0,0 +1,18 @@
+import babel from 'rollup-plugin-babel';
+import autoExternal from 'rollup-plugin-auto-external';
+
+export default {
+  input: 'cache-dir/gatsby-browser-entry.js',
```

Comment:
This will fail, unfortunately. We _want_ the files to remain unbundled so that we can conditionally _require_ certain files.

For instance, if we take [`cache-dir/public-page-renderer`](https://github.com/gatsbyjs/gatsby/blob/7405585f5a98231c3391dda18d8425300f9dcde8/packages/gatsby/cache-dir/public-page-renderer.js) we can see the various conditional require statements.

Bundling this for the browser honestly just wouldn't work, I don't think, because it'll trace those dependencies, exclude conditional requires that we'd want in certain phases, and then create a (smaller, but not as functional) bundle file.

Reply:
fair point. i'm not familiar enough with the approach, so this is some of the detail i was hoping to tease out. there could possibly still be a way to accomplish as a bundle (or multiple), but that could be more of an investment than its worth at this point. 

maybe the better approach would be simple transpilation then, but with `modules` set to `false` for `@babel/env`, similar to what was done for the common js version.